### ISSUE TITLE
removing _ActivityAssociatedAgent broke "hints" for Activity.wasAssociatedWith

### ISSUE THREAD

**Issue by bpow** [clingen-data-model/interpretation#47]
@larrybabb : The `Activity` sheet had referred to `_ActivityAssociatedAgent` to populate the greyed-out "hints" in the columns `wasAssociatedWith[0]`, etc.

Those columns are not used downstream (the join is performed in ruby code), so they are not strictly necessary. My ArrayFormula-fu is not good enough to figure out if the columns can be fixed (would have to index in to `activityAttribute` where attributeId (second column) is A134 and appropriate activity...)

TL;DR-- I would be fine with just deleting those columns from the `Activity` sheet.

**Comment by larrybabb** [comment]
ok by me. feel free to do what you need to.
